Stream_support/package_info/Stream_support/long_description.txt

```diff
@@ -12,7 +12,7 @@ Stream_support Package
     -  CGAL/IO/Verbose_ostream.h
 
          The class CGAL_Verbose_ostream is useful for debugging and
```

Comment:
```suggestion
         The class `CGAL_Verbose_ostream` is useful for debugging and
```

Reply:
As far as I can tell this file is containing a general description and is not used in the CGAL documentation.
As it is not used inside the CGAL documentation I don't think it is wise to add the backticks (they should otherwise also be added elsewhere in this file).